content/blog/2021-02-01-tidynhl-analyse/index.fr.md

```diff
@@ -0,0 +1,711 @@
+## 6703:   8478499      Adin Hill  20182019      2018-19     regular      53               ARI           NA           NA             NA            NA               NA         NA         NA            NA         NA         NA           NA             NA          NA              NA                NA               NA            NA              NA                NA               NA            NA              NA                NA               NA            NA           13             11           7             5           0         0               1                322                 32      0.9006211   2.757234   696.3500                   271                    27         0.9003690                    41                     5         0.8780488                    10                     0                 1
+## 6704:   8478499      Adin Hill  20192020      2019-20     regular      53               ARI           NA           NA             NA            NA               NA         NA         NA            NA         NA         NA           NA             NA          NA              NA                NA               NA            NA              NA                NA               NA            NA              NA                NA               NA            NA           13              9           2             4           0         3               0                343                 28      0.9183673   2.621859   640.7667                   284                    19         0.9330986                    45                     9         0.8000000                    14                     0                 1
+## 6705:   8478916   Joey Daccord  20182019      2018-19     regular       9               OTT           NA           NA             NA            NA               NA         NA         NA            NA         NA         NA           NA             NA          NA              NA                NA               NA            NA              NA                NA               NA            NA              NA                NA               NA            NA            1              1           0             1           0         0               0                 40                  5      0.8750000   5.000000    60.0000                    35                     3         0.9142857                     3                     2         0.3333333                     2                     0                 1
+```
+
+Finalement, nous allons faire quelques transformations et manipulations sur ces
```

Comment:
(pas certain... est-ce que j'ai introduit un anglicisme?)
```suggestion
Finalement, nous allons appliquer quelques transformations et manipulations à ces
```

Reply:
appliquer me le bon mots pour le concept, faire me semble être plus un anglicisme (we will do). Faire me semble être une action de faire alors qu'ici on applique sur des données des transformations.

Reply:
Right ! bon point